bom/runtime/pom.xml

```diff
@@ -173,6 +173,8 @@
         <quarkus-security.version>1.0.1.Final</quarkus-security.version>
         <keycloak.version>8.0.1</keycloak.version>
         <logstash-gelf.version>1.13.0</logstash-gelf.version>
+        <jsch.version>0.1.55</jsch.version>
+        <jzlib.version>1.1.1</jzlib.version>
```

Comment:
Maybe we can add those 2 to dependabot? (It can be done separately)

Reply:
Maybe, but I rather reuse the same version used by JGit